SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py

```diff
@@ -27,6 +31,129 @@ def customise(process):
 
     return process
 
+
+def customise_SimTkOnly(process):
```

Comment:
SimulationTkOnly is indeed useless, I'm currently realizing this. As long as the xml is correctly tweaked there is no need to modify g4simhit config script.

For DigiTkOnly I also need to make some simplification, the customize method is not fully correct. At the end this method will indeed differ only marginally from the customize_digi method

I will push the new scripts today